src/getHumanAge.test.js

```diff
@@ -8,6 +8,30 @@ describe(`Function 'getHumanAge':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(getHumanAge(5, 1)).toBeInstanceOf(Array);
+  });
+
+  it(`should return empty array when input is empty array`, () => {
+    expect(getHumanAge()).toEqual([NaN, NaN]);
+  });
```

Comment:
It's not an empty array. An empty array is `[]`.
You can remove this test, it's not needed here